src/components/ToDoItem.tsx

```diff
@@ -1,14 +1,20 @@
 import React from "react";
 import { Button } from "./Button";
-import { Input } from "./Input";
+import { Input, InputType } from "./Input";
 import { TextWithId } from "../types/common";
+import { text } from "node:stream/consumers";
 
-type ToDoItem = {
+export type ToDoItem = {
   item: TextWithId;
   updateItem: () => void;
   deleteItem: (id: number) => void;
   setNewTextWithId: (item: TextWithId | null) => void;
   newTextWithId: TextWithId | null;
+  className?: string;
+  buttonTextRemove: string;
+  buttonTextSave: string;
+  buttonTextEdit: string;
+  editInputType?: InputType;
```

Comment:
If you do not plan to pass anything here, you could remove it, as this is middle level component and its props should be defined by its purpose